lib/pure/ospaths.nim

```diff
@@ -517,6 +517,7 @@ when declared(getEnv) or defined(nimscript):
     tags: [ReadEnvEffect, ReadIOEffect].} =
     ## Returns the config directory of the current user for applications.
```

Comment:
Please expand on this documentation by stating that the ``XDG_CONFIG_DIR`` env var is consulted on non-Windows OS'.

Make sure to add a new line, like so:

```
## Foo bar
##
## Next line
```